Assets/PC2D/Scripts/PlatformerMotor2D.cs

```diff
@@ -1726,7 +1716,7 @@ private void CheckWallInteractionValidity()
             min.x = min.x - envCheckDistance;
         }
         else if (HasFlag(CollidedSurface.RightWall) &&
-                 _collidedNormals[DIRECTION_RIGHT] == -Vector2.right &&
+                 _collidedNormals[DIRECTION_RIGHT] == Vector2.left &&

```

Comment:
We'll leave this in but Vector2.left was introduced relatively recently and I had originally used them but it broke people on earlier Unity builds. But it's been a few months now.